bftcosi/bftcosi.go

```diff
@@ -110,34 +107,43 @@ type collectStructs struct {
 	tmpMutex sync.Mutex
 	// exceptions given during the rounds that is used in the signature
 	tempExceptions []Exception
+	// respondedNodesPrepare stores a list of nodes that sent a commit
+	// message in the prepare phase
+	respondedNodesPrepare map[kyber.Point]*onet.TreeNode
+	// respondedNodesCommit stores a list of nodes that sent a commit
+	// message in the commit phase
+	respondedNodesCommit map[kyber.Point]*onet.TreeNode
 	// temporary buffer of "prepare" commitments
 	tempPrepareCommit []kyber.Point
 	// temporary buffer of "commit" commitments
 	tempCommitCommit []kyber.Point
 	// temporary buffer of "prepare" responses
 	tempPrepareResponse []kyber.Scalar
-	// temporary buffer of the public keys for nodes that responded
-	tempPrepareResponsePublics []kyber.Point
 	// temporary buffer of "commit" responses
 	tempCommitResponse []kyber.Scalar
+
+	committedInPreparePhase bool
+	committedInCommitPhase  bool
 }
 
 // NewBFTCoSiProtocol returns a new bftcosi struct
-func NewBFTCoSiProtocol(n *onet.TreeNodeInstance, verify VerificationFunction) (*ProtocolBFTCoSi, error) {
+func NewBFTCoSiProtocol(n *onet.TreeNodeInstance, verify VerificationFunction, to time.Duration) (*ProtocolBFTCoSi, error) {
```

Comment:
Here you are changing the signature of the method, which is not allowed during the current development-cycle (no API-changes until September 2018 ;)

Reply:
I'm merging to the byzgen_poc branch. Is this also not allowed?